lib/cfonb/operation_detail/lib.rb

```diff
@@ -3,8 +3,14 @@
 module CFONB
   module OperationDetail
     class LIB
+      ATTRIBUTES = %i[free_label].freeze
+
       def self.apply(operation, line)
-        operation.label += "\n#{line.detail.strip}"
+        operation.free_label = if operation.free_label.nil?
+          line.detail.strip.to_s
+        else
+          "#{operation.free_label}\n#{line.detail.strip}"
+        end
```

Comment:
This doesn't work ?
```suggestion
        operation.free_label += if operation.free_label.nil?
          line.detail.strip
        else
          "\n#{line.detail.strip}"
        end
```

Reply:
I modified it since `free_label` can be nil because it is not part of the base attributes, and then we cannot concatenate the string 😢